giesela/config/app.py

```diff
@@ -25,7 +26,15 @@ class Mongodb(ConfigObject):
     database: str = "Giesela"
 
 
+class LavalinkNodeRegion(enum.Enum):
+    GLOBAL = "global"
+    EU = "eu"  # Eu first, America second
```

Comment:
You know, I would say something here, except *EU* does come before *US* alphabetically, so lets go with that 😂